config/locales/client.en.yml

```diff
@@ -2454,7 +2454,10 @@ en:
       tags: "Tags"
       choose_for_topic: "choose optional tags for this topic"
       delete_tag: "Delete Tag"
-      delete_confirm: "Are you sure you want to delete that tag?"
+      delete_confirm:
+        one: "Are you sure you want to delete this tag and remove it from 1 topic it is assigned to?"
+        other: "Are you sure you want to delete this tag and remove it from {{count}} topics it is assigned to?"
+        zero: "Are you sure you want to delete this tag?"
```

Comment:
It's not possible to use the `zero` key in the English locale. I suggest you use something like `delete_confirm_no_topics` instead.